_posts/2024-04-16-pyos-open-education-resources-announcement.md

```diff
@@ -0,0 +1,99 @@
+
+## pyOpenSci believes in open education resources (OER)
+<figure>
+    <a href="/images/blog/2024/april/pyos-community-dev.png">
+    <img src="/images/blog/2024/april/pyos-community-dev.png" style="max-width:100%" alt="Multiple hands of various skin tones reaching up to a speech bubble that reads 'Community Developed'.">
+    </a>
+    <figcaption>
+      pyOpenSci's open education resources are community-developed
+    </figcaption>
+</figure>
+When pyOpenSci publishes community-developed Python tutorials for scientists, it goes through a rigorous process of review and evaluation. ​​All of our tutorials are created through a multi-stage community review process–where tutorials are first developed by the pyOpenSci team or community members–before being  reviewed by tool maintainers to ensure ideas and concepts are accurate. Tutorials then go through several rounds of community review for accuracy, usability and accessibility before being published as part of pyOpenSci’s open education resources..
+
+All of our written content is available on the [Learn section of our website](https://www.pyopensci.org/learn.html), where you’ll find resources such as our [Python Packaging Tutorial](https://www.pyopensci.org/python-package-guide/tutorials/intro.html#), as well as in-depth guides on [Python packaging](https://www.pyopensci.org/python-package-guide/package-structure-code/intro.html), [documentation](https://www.pyopensci.org/python-package-guide/documentation/index.html), and [testing](https://www.pyopensci.org/python-package-guide/tests/index.html).
+
+### Free, online, asynchronous training
+<figure>
+    <a href="/images/blog/2024/april/pyos-collab-learn.png">
+    <img src="/images/blog/2024/april/pyos-collab-learn.png" style="max-width:100%" alt="A woman and a man sitting across from each other at a high table. They are smiling and working on their laptops. The text reads 'Essential Collaboration Skills for Scientists'.">
+    </a>
+    <figcaption>
+      Essential Collaboration Skills for Scientists is just one of the many free, online trainings we're excited to develop and deliver!
+    </figcaption>
+</figure>
+Our vision for our free, online trainings is for the curriculum to be added to our library of open education resources, making them free, accessible, and published for anyone to use. And we intend to experiment with delivering this curriculum in an asynchronous, cohort-based manner that allows learners from all over the world to come together for a period of time to work through the curriculum together. If you were part of one of the original cohorts of the [R for Data Science Online Learning Community](https://dslc.io/) created by our Community Manager, [Jesse Mostipak](https://github.com/kierisi), then you know that these asynchronous trainings will be thoughtfully crafted, a fantastic learning experience, and a ton of fun!
```

Comment:
I love this as it makes us sound like powerhouse. The only thing is the link to the community will take a user away from our site to another organization not related to us. What about turning it into a href like this:

<a href="https://dslc.io/" target="_blank">R for Data Science Online Learning Community</a> 

NOTE: please check in CI that it works but i think it should.that way the other links that take folks away are still related to us.

Reply:
looks like it worked!